### PR TITLE
[CoSim improving check

### PR DESCRIPTION
**Pull request by philbucher**
otherwise the domain-size warning is issued twice if no domain-size is specified

### CODE REVIEW

File: applications/CoSimulationApplication/python_scripts/coupling_interface_data.py [hunk 1]
```diff
@@ -70,19 +70,20 @@ def Initialize(self):
             else:
                 if self.variable_type == "Array" and self.dimension not in [1,2,3]:
                     raise Exception('"dimension" can only be 1,2,3 when using variables of type "Array"')
-                domain_size = self.GetModelPart().ProcessInfo[KM.DOMAIN_SIZE]
-                if domain_size == 0:
-                    cs_tools.cs_print_warning('CouplingInterfaceData', 'No "DOMAIN_SIZE" was specified for ModelPart "{}"'.format(self.GetModelPart().Name))
-                if domain_size != self.dimension:
-                    cs_tools.cs_print_warning('CouplingInterfaceData', '"DOMAIN_SIZE" ({}) of ModelPart "{}" does not match dimension ({})'.format(domain_size, self.GetModelPart().Name, self.dimension))
+                if not KM.DOMAIN_SIZE in self.GetModelPart().ProcessInfo:
+                    cs_tools.cs_print_warning('CouplingInterfaceData', 'No "DOMAIN_SIZE" was specified for ModelPart "{}"'.format(self.model_part_name))
```
Comment (adityaghantasala) [inline applications/CoSimulationApplication/python_scripts/coupling_interface_data.py:74]: Shouldn't this be an error ?

Comment (philbucher) [reply applications/CoSimulationApplication/python_scripts/coupling_interface_data.py:74]: not all modelparts (e.g. in DEM) declare a domain size, so for now I would leave it as a warning
Also for external solvers we will have to think how to do it

Comment (adityaghantasala) [reply applications/CoSimulationApplication/python_scripts/coupling_interface_data.py:74]: ok ... DEM ... 
Also if that is a external solver ... we can specify it in JSON when defining the solver. We can discuss it in person .. for now we can go forward.

**Review by adityaghantasala — APPROVED** [review]

File: applications/CoSimulationApplication/python_scripts/coupling_interface_data.py (hunk 1 above)
Comment (philbucher) [reply applications/CoSimulationApplication/python_scripts/coupling_interface_data.py:74]: ok sounds good

**Pull request merged**